include/DDpackage.h

```diff
@@ -71,6 +71,20 @@ namespace dd {
         ListElementPtr next;
         int            w;
     };
+    struct Control {
+        enum controlType : bool { pos = true,
+                                  neg = false };
+
+        unsigned short qubit = 0;
+        controlType    type  = pos;
+
+        explicit Control(unsigned short qubit = 0, controlType type = pos):
+            qubit(qubit), type(type){};
+
+        static bool compare(Control c1, Control c2) {
+            return c1.qubit < c2.qubit;
```

Comment:
Where do we use the `compare` function? (Is it for sets? This should also consider `the controlType` to have an unambiguous order for non-equal instances.)

Reply:
I guess the intent is to use it for sorting the vector of controls at some point.
Probably should be implemented as an overload to `operator<` so that `std::set` can directly use it to compare controls.
I believe an overload to `operator==` might also be needed for `std::set` to work.